frontend/src/static/js/v_authorship.js

```diff
@@ -199,6 +199,7 @@ window.vAuthorship = {
       this.isLoaded = true;
 
       this.activeFilesCount = this.selectedFiles.length;
+      this.sortFiles();
```

Comment:
this should be able to be removed since the value of toReverseSortFiles is watched

Reply:
We cannot remove it because even if `toReverseSortFiles` is watched, there is nothing to activate it. So, the default sorting is still ascending order, which defeats the purpose of the PR. Please refer to the screenshot below.

![Screen Shot 2019-11-20 at 8 12 55 PM](https://user-images.githubusercontent.com/35946746/69238214-6b552000-0bd2-11ea-9e57-72eeae18eb46.png)

Reply:
@Tejas2805 I am going to let you figure this out yourself so that maybe you will understand how vue works better this way.

Think about why there isn't a need to add the function there in the first place, yet the files will came out in an arranged order.

Reply:
I know that cause `toReverseSortFiles` is watched, whenever we change the sorting order, it will automatically cause the files to sort themselves, based on the value in `toReverseSortFiles` since its being `watched` but won't we have to activate it the first time to sort the files in required order? I tried running the program by removing `sortFiles()`  and the files came in ascending order with `order` as Descending.
So I am confused as to whether you are saying that we shouldn't need `sortFiles()` at all, or should I call the function somewhere else.

Reply:
>So I am confused as to whether you are saying that we shouldn't need sortFiles() at all, or should I call the function somewhere else.

Yes, you dont need sortFiles at all, you just need to change some code somewhere to make the default descending.

Reply:
@eugenepeh I have made the change. Wished to understand something- Is this way preferred cause this reduces the call stack?

Reply:
The original implementation will increase one additional unnecessary event loop of Vue, which is quite costly if there are multiple files.